schemas/com.snowplowanalytics.monitoring.batch/load_succeeded/jsonschema/3-0-1

```diff
@@ -0,0 +1,192 @@
+              "additionalProperties": false
+            }
+          ]
+        },
+        "timestamps": {
+          "description": "Set of timestamps associated with the batch",
+          "type": "object",
+          "properties": {
+            "jobStarted": {
+              "description": "Time when the batch started being shredded",
+              "type": "string",
+              "format": "date-time"
+            },
+            "jobCompleted": {
+              "description": "Time when the batch shredding has been finished (and the message being prepared)",
+              "type": "string",
+              "format": "date-time"
+            },
+            "min": {
+              "description": "The earliest collector_stamp available in the batch",
+              "type": ["string", "null"],
+              "format": "date-time"
+            },
+            "max": {
+              "description": "The latest collector_tstamp available in the batch",
+              "type": ["string", "null"],
+              "format": "date-time"
+            }
+          },
+          "required": ["jobStarted", "jobCompleted", "min", "max"]
+        },
+        "processor": {
+          "description": "Identificator of a shredder sent the message",
+          "type": "object",
+          "properties": {
+            "artifact": {
+              "description": "Name of the artifact",
+              "type": "string",
+              "maxLength": 64
+            },
+            "version": {
+              "description": "Semantic Version of the artifact",
+              "type": "string",
+              "maxLength": 16
+            }
+          },
+          "required": ["artifact", "version"]
+        },
+        "count": {
+          "description": "Count of events in the batch, null means the count could not be calculated",
+          "type": ["object", "null"],
+          "properties": {
+            "good": {
+              "description": "Amount of good events in the batch",
+              "type": "integer",
+              "minimum": 0
+            }
+          }
+        }
+      },
+      "required": ["base", "compression", "typesInfo", "timestamps", "processor"],
+      "additionalProperties": false
+    },
+    "application": {
+      "description": "Loader's name and version",
+      "type": "string",
+      "maxLength": 128
+    },
+    "attempt": {
+      "description": "How many attempts were taken to load the batch",
+      "type": "integer",
+      "minimum": 0
+    },
+    "loadingStarted": {
+      "description": "Time when loader started loading (first attempt)",
+      "type": "string",
+      "format": "date-time"
+    },
+    "loadingCompleted": {
+      "description": "Time when loader finished loading (should match the ingestion timestamp in manifest)",
+      "type": "string",
+      "format": "date-time"
+    },
+    "recoveryTableNames": {
+      "description": "List of recovery table names loaded in this batch",
+      "type": "array",
```

Comment:
As this is a new-field doesn't this need to be `nullable` or does `array` type implictly imply optionality?